create-job-ui/src/components/ListOfOptions.js

```diff
@@ -0,0 +1,47 @@
+import React,{useState} from 'react'
+
+export const ListOfOptions = ({items,setItems,options,setOptions}) => {
+
+//const [items, setItems] = useState([]);
+const [selectedItem, setSelectedItem] = useState('');
+
+//const options = ["Option 1", "Option 2", "Option 3"];
+//const [options, setOptions] = useState(['Select Option', 'Option 1', 'Option 2']);
+const [newOption, setNewOption] = useState('');
+
+function handleSubmit(e) {
+    e.preventDefault();
+    if(options.indexOf(newOption) === -1 && newOption!==""){
+      setOptions([...options, newOption]);
+      setNewOption('');
+    }else{
+      console.log(`${newOption} already exists in the options`);
+    }
+  }
+
+
+const handleSelectChange = (event) => {
```

Comment:
@asharMehmood2211004KHICND we could use `useCallback` for these so that the functions aren't reevaluated on each render